content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -1443,3 +1443,48 @@ SELECT * from dolt_constraint_violations_child;
 +----------------+----+-----------+
 */
 ```
+
+# Access Control
+Dolt stored procedures are access controlled using the GRANT permissions system. Dolt deviates slightly from the common permissions model due to the fact that some dolt procedures need to be more tightly controlled for safety of the server.
+
+User's who need common Dolt capability such as adding and committing to a branch will need Execute permission granted on the database in question. As a privileged user, you can grant access with the following command:
+
+```sql
+mydb> GRANT EXECUTE ON mydb.* TO pat@localhost
+```
+
+This will give the user, Pat, the ability run all stored procedures on the database. This includes Dolt procedures as well as user defined procedures. If you need to use fine grained permissions, you can grant them individually:
```

Comment:
```suggestion
This will give the user, `pat`, the ability run all stored procedures on the database. This includes Dolt procedures as well as user defined procedures. If you need to use fine-grained permissions, you can grant them individually:
```